dev_guide/builds/build_inputs.adoc

```diff
@@ -333,14 +333,23 @@ A URI pattern must consist of:
 
 In all of the above, a `*` character is interpreted as a wildcard.
 
-[NOTE]
+[IMPORTANT]
 ====
 URI patterns only match Git source URIs which are conformant to RFC3986. For example,
-https://github.com/openshift/origin.git. They do not match the alternate SSH
-style that Git also uses. For example, git@github.com:openshift/origin.git.
+`https://github.com/openshift/origin.git`. They do not match the alternate SSH
+style that Git also uses. For example, `git@github.com:openshift/origin.git`.
 
 It is not valid to attempt to express a URI pattern in the alternate style, or
 to include a username/password component in a URI pattern.
+
+For example, if you use `ssh://git@bitbucket.atlassian.com:7999/ATLASSIAN/jira.git` for a git repository URL, the source secret must be specified as `pass:c[ssh://bitbucket.atlassian.com:7999/*]` (and not `pass:c[ssh://git@bitbucket.atlassian.com:7999/*]`).
+
```

Comment:
There were formatting errors without `pass:c[ ]`. It was treating the content between *'s as bold.